.github/workflows/aws_tfhe_signed_integer_gpu_tests.yml

```diff
@@ -16,10 +16,17 @@ on:
   # Allows you to run this workflow manually from the Actions tab as an alternative.
   workflow_dispatch:
   pull_request:
+  push:
+    branches:
+      - main
```

Comment:
test on push to main ?

Reply:
why ?

Reply:
I think those we want to run on approve, we have the H100 tests for signed & unsigned operations that should run on push to the PRs